Example/Podfile.lock

```diff
@@ -45,4 +45,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 591559c46a2b0a49e687795b8ae46fadbddf8fd4
 
-COCOAPODS: 1.1.1
+COCOAPODS: 1.2.0.beta.1
```

Comment:
Let's not use the beta version for now, I think it is better to keep 1.1.1 which is the current stable.